src/databricks/labs/remorph/assessments/scheduler/install/README.md

```diff
@@ -0,0 +1,24 @@
+# Installing the Usage Collection Scheduler
+
```

Comment:
moving the markdown inside 

`/docs/remorph/docs/assessment` folder will help for auto document generation

Reply:
Great idea 👍